lib/approvals/writer.rb

```diff
@@ -16,15 +16,20 @@ module Writer
       html: Writers::HtmlWriter.new,
       hash: Writers::HashWriter.new,
       array: Writers::ArrayWriter.new,
+      txt: Writers::TextWriter.new,
     }
 
 
     class << self
       def for(format)
-        if REGISTRY.include?(format)
-          REGISTRY[format]
-        else
-          TextWriter.new
+        begin
+          REGISTRY[format] || Object.const_get(format).new
+        rescue NameError => e
+          error = ApprovalError.new(
+            "Approval Error: #{ e }. Please define a custom writer as outlined"\
+            " in the 'Customizing formatted output' documentation."
+          )
```

Comment:
Nice! I wonder if we should say where that documentation is. Maybe link to the README online?